kafka/metadata.csv

```diff
@@ -1,31 +1,42 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
-kafka.net.bytes_out.rate,gauge,10,byte,second,Outgoing byte rate.,0,kafka,bytes out
-kafka.net.bytes_in.rate,gauge,10,byte,second,Incoming byte rate.,0,kafka,bytes in
-kafka.net.bytes_rejected.rate,gauge,10,byte,second,Rejected byte rate.,-1,kafka,bytes rejected
```

Comment:
Can we change these? The submission type was always a rate

Reply:
We **can** change this because the submission type on the intake takes precedence over the metadata type if it's different.
But I don't think that's true here, the submitted type seems to be a `gauge` in-app. It's just computed as a rate no?